src/main/java/com/github/zetten/maven/xvfb/AbstractXvfbMojo.java

```diff
@@ -128,17 +136,44 @@
 	protected String fbdir;
 
 	/**
+	 * The number of seconds to wait for return value when stopping Xvfb process.
+	 */
+	@Parameter(defaultValue = "13", required = false)
+	protected long stopTimeoutInSeconds;
+
+	/**
+	 * Additional arguments to the Xvfb process.
+	 */
+	@Parameter(required = false)
+	protected List<String> xvfbArgs;
```

Comment:
That will be difficult to pass an array via a property, no?
There could be an additional parameter, with property, for a line of arguments.